src/api/updateUserDetails.ts

```diff
@@ -0,0 +1,19 @@
+// utils/updateUserDetails.js
+import axios from 'axios';
+
+const updateUserDetails = async (formData: any) => {
```

Comment:
if you are using this wiith tankstack/react-query, convert this to a promise, you can look into other functions for reference